runtime/src/serde_snapshot/tests.rs

```diff
@@ -261,7 +261,7 @@ mod test_bank_serialize {
 
     // These some what long test harness is required to freeze the ABI of
     // Bank's serialization due to versioned nature
-    #[frozen_abi(digest = "8bNY87hccyDYCRar1gM3NSvpvtiUM3W3rGeJLJayz42e")]
+    #[frozen_abi(digest = "ByqDP7c1XGQSnrVsTCweFcFtyTG6fyohueFCXLBPEMeC")]
```

Comment:
Any backwards compatibility concerns from changing the serialization hash here? Or the fields are serde_skip?

Reply:
I am deriving Serialize/Deserialize manually for the new `VoteAccounts` type which I think will provide backward compatibility. I will tag you on the related section of the code to take a look.